README.rst

```diff
@@ -26,18 +26,82 @@
 .. image:: https://img.shields.io/badge/-PyScaffold-005CA0?logo=pyscaffold
     :alt: Project generated with PyScaffold
     :target: https://pyscaffold.org/
+.. image::https://img.shields.io/badge/python-3.9+-blue.svg
+    :target: https://www.python.org/downloads/
 
 |
 
 ===================
-flexmeasures-client
+FlexMeasures Client
 ===================
 
 
-    Add a short description here!
+The FlexMeasures Client provides a python package to connect to a `FlexMeasures <https://github.com/FlexMeasures/flexmeasures>`_ server to manage flexible assets.
+
+The Flexmeasures Client package provides functionality for authentication, posting sensor data, triggering schedules and retrieving schedules from a FlexMeasures instance through the API.  
+
+
+Getting Started
+===============
+
+To get started using the FlexMeasures Client package first an account needs to be registered with a FlexMeasures instance or a local FlexMeasures instance needs to be created. Registring a to a FlexMeasures instance can be done through `Seita BV <https://seita.nl/>`_. To create a local instance of FlexMeasures follow the `FlexMeasures documentation <https://flexmeasures.readthedocs.io/en/latest/index.html>`_. 
+
+In this example we are connecting to ``localhost:5000``, To connect to a different host add the host in the initialization of the client.
+
+Install using ``pip``::
```

Comment:
only one `:`, I think. Likewise below.

Reply:
This seems to correctly show in the readme on github. Please let me know if there is another reason that it should be 1 :